docs/cody/clients/install-vscode.mdx

```diff
@@ -311,9 +311,22 @@ Cody also works with Cursor, Gitpod, IDX, and other similar VS Code forks. To ac
 
 ## Supported LLM models
 
-Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody **Pro** can choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Haiku, Opus and Sonnet), Claude 3.5 Sonnet, ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, GPT-4o, Mixtral, Google Gemini 1.5 Pro and Google Gemini 1.5 Flash.
-
-![LLM-models-for-cody-pro](https://storage.googleapis.com/sourcegraph-assets/Docs/vscode-llm-models-june20-2024.png)
+Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody **Free** and **Pro** can choose from a list of supported LLM models for Chat and Commands. Here's a preview:
+
+|   **Cody Free**    |           **Cody Pro**            |
+| ------------------ | --------------------------------- |
+| Claude 3 Sonnet    | Claude Instant                    |
+| Claude 3.5 Sonnnet | Claude 2.0                        |
+| Claude 3 Haiku     | Claude 3 (Haiku, Opus and Sonnet) |
+| GPT 3.5 Turbo      | Claude 3.5 Sonnet                 |
+| Gemini 1.5 Flash   | GPT 3.5 Turbo                     |
+| Gemini 1.5 Pro     | GPT 4 Turbo Preview               |
+| Mixtral 8x22B      | GPT-4o                            |
+| Mixtral 8x7B       | Mixtral                           |
+| Mixtral 8x7B       | Gemini 1.5 Pro                    |
+| Mixtral 8x7B       | Gemini 1.5 Flash                  |
```

Comment:
i think this is not clear that Cody Pro includes models from Cody Free because some overlaps are included and some are not. Maybe we should just do the ✅ for each models instead?

Reply:
Nice call.